### PR TITLE
depext: fix Debian/apt lookup for installed packages

### PR DESCRIPTION
**Pull request by rjbou**
dpkg-query output have the information in the first column, but only the second character needed to be looked up.

### CODE REVIEW

File: master_changes.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ users)
 ## Opamfile
 
 ## External dependencies
+  * [BUG] Fix apt/debian lookup for installed packages [#XXX @rjbou]
```
Comment (dra27) [inline master_changes.md:62]:
```suggestion
  * [BUG] Fix apt/debian lookup for installed packages [#6054 @rjbou]
```

File: src/state/opamSysInteract.ml [hunk 2]
```diff
@@ -774,12 +774,13 @@ let packages_status ?(env=OpamVariable.Map.empty) config packages =
     let get_installed str_pkgs =
       (* ouput:
          >ii  uim-gtk3                 1:1.8.8-6.1  amd64    Universal ...
-         >ii  uim-gtk3-immodule:amd64  1:1.8.8-6.1  amd64    Universal ...
+         >ri  uim-gtk3-immodule:amd64  1:1.8.8-6.1  amd64    Universal ...
```
Comment (dra27) [inline src/state/opamSysInteract.ml:777]: Worth expanding this a bit (looking at the `dpkg-query` man page) that the first code can be `[uihrp]` but that we only care about the second character?

**Review by dra27 — APPROVED** [review]
I'd possibly expand the comment to be more explicit that we're ignoring the first character (and why), but it was very easy to answer that from the dpkg-query man page!

File: src/state/opamSysInteract.ml (hunk 2 above)
Comment (dra27) [reply src/state/opamSysInteract.ml:777]: New comment added below looks perfect, thanks!

**Comment by kit-ty-kate** [comment]
Queued on https://github.com/ocaml/opam/pull/6063 for CI to succeed

**Pull request merged**